src/SqlSquare/Lenses.purs

```diff
@@ -0,0 +1,232 @@
+module SqlSquare.Lenses where
+
+import Prelude
+
+import Data.Lens (Prism', prism', Lens', lens, Iso', iso)
+import Data.List as L
+import Data.Maybe as M
+import Data.Newtype (class Newtype, wrap, unwrap)
+import Data.NonEmpty as NE
+
+import Matryoshka (class Recursive, class Corecursive, embed, project)
+
+import SqlSquare.AST as S
+import SqlSquare.Utils (type (×), (∘), (⋙))
+
+_Newtype ∷ ∀ n t. Newtype n t ⇒ Iso' n t
+_Newtype = iso unwrap wrap

```

Comment:
There's already an iso for this: https://pursuit.purescript.org/packages/purescript-profunctor-lenses/2.6.0/docs/Data.Lens.Iso.Newtype#v:_Newtype